<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::ProfileDialog</class>
 <widget class="QDialog" name="Core::ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="logoLabel">
     <property name="pixmap">
      <pixmap resource="../core.qrc">:/core/images/logo.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="profilesGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QTreeWidget" name="profileTree">
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerPixel</enum>
        </property>
        <property name="indentation">
         <number>0</number>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string notr="true"/>
         </property>
        </column>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="passwordHorizontalLayout">
        <item>
         <widget class="QLabel" name="passwordLabel">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="infoHorizontalLayout">
     <item>
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string notr="true">lily.im v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="websiteLabel">
       <property name="text">
        <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a href=&quot;http://www.lily.im/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.lily.im&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>passwordEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profileTree</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>Core::ProfileDialog</receiver>
   <slot>loginProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>27</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>Core::ProfileDialog</receiver>
   <slot>loginProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>Core::ProfileDialog</receiver>
   <slot>deleteProfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profileTree</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>Core::ProfileDialog</receiver>
   <slot>updateControls(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateControls(QTreeWidgetItem*)</slot>
  <slot>loginProfile()</slot>
  <slot>deleteProfile()</slot>
 </slots>
</ui>
